pingdom-checks/appeals.ping

```diff
@@ -31,5 +31,5 @@ pingdom save-check \
   -a url="/services/appeals/v0/healthcheck" \
   -a apikey="$SANDBOX_APPEALS_API_KEY" \
   -a group="appeals" \
-  -a userids_csv="$NO_USERS" \
+  -a userids_csv="$STATUSPAGE_SANDBOX_APPEALS_STATUS" \
```

Comment:
Do these dev checks need to be sent to statuspage? We're just going to remove them within the next sprint or 2 no?
In addition, this being sent to the sandbox status page address just doubles up on health checks being sent right?

Reply:
The scope of the story was to update the IDs, so I'm not sure.

Reply:
The scope of the story is 
> Integrate Pingdom and Statuspage for this API in Staging and Production environments